tools/lsp/server_loop.rs

```diff
@@ -35,7 +35,7 @@ const QUERY_PROPERTIES_COMMAND: &str = "queryProperties";
 fn command_list() -> Vec<String> {
     let mut result = vec![];
 
-    #[cfg(any(feature = "preview", target_arch = "wasm32"))]
+    #[cfg(any(feature = "preview", feature = "preview-lense"))]
```

Comment:
Since preview implies preview-lense, it can be simplified:
```suggestion
    #[cfg(feature = "preview-lense")]
```